iotdb-core/datanode/src/main/java/org/apache/iotdb/db/protocol/client/ConfigNodeClient.java

```diff
@@ -243,6 +226,15 @@ private void tryToConnect() throws TException {
         logger.warn("The current node may have been down {},try next node", configLeader);
         configLeader = null;
       }
+    } else {
+      try {
+        // Wait to start the next try
+        Thread.sleep(RETRY_INTERVAL_MS);
+      } catch (InterruptedException e) {
+        Thread.currentThread().interrupt();
+        throw new TException(
```

Comment:
Seems not necessary to throw an exception here ?

Reply:
fixed